[Docs](bangc-ops): Update docs for v0.8.0 (#793)
Cambricon/mlu-ops: docs/bangc-docs/api_guide/update.rst

This section lists contents that were made for each product release.

* V0.8.0

  **Date:** August 9, 2023

  **Changes:**

  - Added the following new operations:

>     - border_align

Reviewer: 1

This section lists contents that were made for each product release.

* V0.8.0

  **Date:** August 9, 2023

  **Changes:**

  - Added the following new operations:

    - border_align_backward
    - border_align_forward
    - masked_col2im_forward
    - masked_im2col_forward
    - tin_shift_backward
    - tin_shift_forward

>   - Fixed some bugs:

Reviewer 2: 这行删掉？

>     - dynamic_point_to_voxel_backward: Fix "Buffer Overflow" when compiling with memcheck option.

Reviewer 2: 这部分再确认一下吧，此处只写文档的更新，bug 修复是不是写到release note里？

>     - roi_crop_forward/roi_crop_backward: Fix wrong description of return-value in mlu_op.h.

Reviewer 2: 描述优化一下？ 是这个意思吗？
```suggestion
    - Optimized the description of return value of ``roi_crop_forward`` and ``roi_crop_backward``.

```
